Rebus.RabbitMq/Properties/AssemblyInfo.cs

```diff
@@ -18,7 +18,7 @@
 // to COM components.  If you need to access a type in this assembly from 
 // COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible(false)]
-[assembly: CLSCompliant(true)]
+[assembly: CLSCompliant(false)]
```

Comment:
I assume this is because `SslOptions` is using unsigned primitive types? 

I am not too happy about this particular change. In a recent PR from @K3llr he was initially forced to do the same because one of RabbitMQ's `ushort`s (or something) got exposed, but he solved it by making a CLS compliant data structure to copy the data into instead.

Could the same approach maybe be used here? I.e. essentially making a copy of the `SslOptions` data structure, only using CLS compliant types....